Source/XUnitExtensions/Lit/LitTestCase.cs

```diff
@@ -58,27 +63,40 @@ public class LitTestCase {
 
     public void Execute(ITestOutputHelper outputHelper) {
       Directory.CreateDirectory(Path.Join(Path.GetDirectoryName(filePath), "Output"));
+      // For debugging. Only printed on failure in case the true cause is buried in an earlier command.
+      List<(string, string)> results = new();
 
       foreach (var command in commands) {
         int exitCode;
         string output;
         string error;
         try {
-          outputHelper.WriteLine($"Executing command: {command}");
+          var commandStr = $"{command}";
+          outputHelper.WriteLine($"Executing command: {commandStr}");
```

Comment:
Does that fix something?

Reply:
No fix here, only explicit the string cast conversion which used to happen implicitly before, and enable to debug it more easily - that's how I spotted virtually all the problems with the backslashes in commands.

Reply:
Unless you want otherwise, I'm leaving it because it will ease maintainability in the future as we will add more various tests.

Reply:
Just out of curiosity, how did it help you?

Reply:
I put a breakpoint on line `outputHelper.WriteLine($"Executing command: {commandStr}");` and figure out what the command is immediately by looking at the variable value, and thus I can enter the command execution if I want to inspect it.

Reply:
I see, but can't you just have the breakpoint on the next line? Anyway, it's fine if think that it's worth it.

Reply:
That's true. Historically, I did not find where the output was being written to so but now I do, so let me remove this line.

Reply:
I think we could extract variables almost everywhere in the code to improve the debug experience, so we shouldn't commit those extracted variables to source after we're done debugging. I see you've reverted this change so I'll mark as resolved.